examples/debug/debug.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# This script launches a job pod in a persistent state using 'sleep infinity', 
+# allowing us to utilize the Kubespray environment within this pod for debugging Ansible playbooks.
+# 
+# 1. apply kubean custom resources and create pod.
+# ./debug.sh
+#
+# 2. enter the kubespray container environment.
+# kubectl -n kubean-system exec -it kubean-debug-install-ops-job-xxxxx bash
+#
+# 3. get kubespray-related operation commands inside the container.
+# cat /bin/entrypoint.sh
+#
+# 4. delete custom resources.
+# kubectl delete cluster debug
+#
+
+DEBUG_NODE_ADDR="172.168.10.66"
+DEBUG_NODE_USER="root"
+DEBUG_NODE_PASS="1q2w3e4r"
+
+KUBEAN_VERSION="v0.13.4"
```

Comment:
It should be renamed as `SPRAY_JOB_VERSION`

Reply:
At the same time, these parameters can be considered to accept external environment variables.